front-end/src/components/Banner/Banner.jsx

```diff
@@ -0,0 +1,23 @@
+import './Banner.css';  
+import { useState } from 'react';    
+import { banner112022, banner042023, banner112023 , banner102024 } from '../../assets/banners';
+
+const banners = [banner102024, banner112023, banner042023, banner112022];  
+
+export const Banner = () => {  
+
+    return (  
+        <div className="banner-section">  
+            <div className="banner-list">  
+                <div className="banner-list__previous"></div>
```

Comment:
Is there any special reason for this className, if not should name in kebab case